src/migration-wizard/steps/connect/sources-table/empty-state/EmptyState.tsx

```
@@ -1,23 +1,19 @@
-/* eslint-disable simple-import-sort/imports */
 import React, { useCallback, useState } from 'react';
-
 import {
-  Alert,
   Button,
+  EmptyState as PFEmptyState,
   EmptyStateActions,
```

ESLint

[error] Member 'EmptyStateActions' of the import declaration should be sorted alphabetically.

(sort-imports)